clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/security/DockerRegistryNamedAccountCredentials.groovy

```diff
@@ -124,6 +139,7 @@ public class DockerRegistryNamedAccountCredentials implements AccountCredentials
   private final String environment
   private final String accountType
   private final String address
+  private final String registry
```

Comment:
This is groovy, man. Drop the `private`s and the getter/setters ;-) 

![](http://4.bp.blogspot.com/_0M_2rFZRIug/S961AeNOVHI/AAAAAAAACUE/tftDgYrUvEo/s400/groovy.gif)